src/services/match-experience.service.ts

```diff
@@ -34,6 +36,79 @@ class MatchExperienceService {
             throw error;
         }
     };
+
+    getAllMatchExperiences = async (page: number, limit: number, sortBy: string = "date") => {
```

Comment:
```suggestion
   type SortByExperiences = "date" | "likes"
    getAllMatchExperiences = async (page: number, limit: number, sortBy: SortByExperiences = "date") => {
```
for better type safety. that makes sure you can not pass other string